packages/platform-android/src/commands/runAndroid/__tests__/runOnAllDevices.test.ts

```diff
@@ -22,6 +22,7 @@ describe('--appFolder', () => {
   });
 
   it('uses task "install[Variant]" as default task', () => {
+    // @ts-ignore
```

Comment:
this is JS file, why it needs ts-ignore?

Reply:
This is a TS file. Check again C: